test/test_euler.py

```diff
@@ -377,6 +406,8 @@ def test_facial_flux(actx_factory, order, dim):
         iff_split = split_conserved(dim, interior_face_flux)
         assert fnorm(iff_split.mass) < tolerance
         assert fnorm(iff_split.energy) < tolerance
+        if nspecies > 1:
+            assert fnorm(iff_split.massfractions) < tolerance
```

Comment:
We should make it so that this succeeds (i.e. the norm returns 0) for an empty object array, to avoid the branch.